composer.json

```diff
@@ -14,7 +14,7 @@
         "php": ">=5.3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "~4 | ~5",
+        "phpunit/phpunit": "^4.8.35 | ^5.7",
```

Comment:
Am I right that you're using such specific versions because it's only in these (and above) that the PHPUnit files are in namespaces?

Reply:
yeah, the forward compatibility layer was backported in the 4.8 branch for the 4.8.35 release. Older 4.8 versions don't have it.

Technically, PHPUnit 5.5+ has the new classes, but anything before 5.7 in the 5.x branch is unmaintained anyway.

Reply:
thanks for answering @stof

Reply:
Thanks both.